nautobot/project-static/admin/css/admin-inlines.css

```diff
@@ -0,0 +1,2 @@
+.sortable-placeholder {border: 1px dashed #ccc;}
```

Comment:
Is this file used anywhere? I don't see it mentioned in the diffs and it doesn't seem to be replacing an existing file in Django.